docs/source/deployment/amazon_emr_serverless.md

```diff
@@ -0,0 +1,244 @@
+
+- Packaging the Kedro project to install it on all EMR Serverless worker nodes. `kedro package` can be used for this, and the resultant `.whl` file is used for installation.
+- Using a custom Python version instead of the default Python installed on EMR Serverless. In the example, [pyenv](https://github.com/pyenv/pyenv) is used for installing the custom Python version.
+- Running a job on EMR Serverless specifying Spark properties. This is needed to use the custom Python and provide an entrypoint script that accepts command line arguments for running Kedro.
+
+Here is an example Dockerfile that can be used:
+
+```shell
+FROM public.ecr.aws/emr-serverless/spark/emr-6.10.0:latest AS base
+
+USER root
+
+# Install Python build dependencies for pyenv
+RUN yum install -y gcc make patch zlib-devel bzip2 bzip2-devel readline-devel  \
+        sqlite sqlite-devel openssl11-devel tk-devel libffi-devel xz-devel tar
+
+# Install git for pyenv installation
+RUN yum install -y git
+
+# Add pyenv to PATH and set up environment variables
+ENV PYENV_ROOT /usr/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+ENV PYTHON_VERSION=3.9.16
+
+# Install pyenv, initialize it, install desired Python version and set as global
+RUN curl https://pyenv.run | bash
+RUN eval "$(pyenv init -)"
+RUN pyenv install ${PYTHON_VERSION} && pyenv global ${PYTHON_VERSION}
+
+# Copy and install packaged Kedro project
+ENV KEDRO_PACKAGE <PACKAGE_WHEEL_NAME>
+COPY dist/$KEDRO_PACKAGE /tmp/dist/$KEDRO_PACKAGE
+RUN pip install --upgrade pip && pip install /tmp/dist/$KEDRO_PACKAGE  \
+    && rm -f /tmp/dist/$KEDRO_PACKAGE
+
+# Copy and extract conf folder
+ADD dist/conf.tar.gz /home/hadoop/
+
+# Initialize log files
+RUN mkdir -p /home/hadoop/logs
+RUN touch /home/hadoop/logs/info.log
+RUN touch /home/hadoop/logs/errors.log
+RUN chmod -R a+w /home/hadoop/logs
+
+# EMRS will run the image as hadoop
+USER hadoop:hadoop
+```
+
+Make sure to replace `<PACKAGE_WHEEL_NAME>` with your own `.whl` file name.
+Here is the `entrypoint.py` entrypoint script:
+
+```python
+import sys
+from <PACKAGE_NAME>.__main__ import main
+
+main(sys.argv[1:])
+
+```
+
+Replace `<PACKAGE_NAME>` with your package name.
+
+### Resources
+For more details, refer to the following resources:
+
+- [Package a Kedro project](https://docs.kedro.org/en/stable/tutorial/package_a_project.html#package-a-kedro-project)
+- [Run a packaged project](https://docs.kedro.org/en/stable/tutorial/package_a_project.html#run-a-packaged-project)
+- [Customizing an EMR Serverless image](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/application-custom-image.html)
+- [Using custom images with EMR Serverless](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/using-custom-images.html)
+
+## Setup
+
+### Prerequisites
+You must create an S3 bucket to store data for EMR Serverless.
```

Comment:
This looks a bit weird when rendered, but I guess it's how our docs CSS work.

![image](https://github.com/kedro-org/kedro/assets/18221871/2747e416-3010-4699-a89e-36ae4f388537)

Reply:
This now looks good in the rendered docs, so resolved now?